include/picongpu/plugins/binning/FilteredSpecies.hpp

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2023-2025 Tapish Narwal
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <pmacc/attribute/FunctionSpecifier.hpp>
+
+namespace picongpu
+{
+    namespace plugins::binning
```

Comment:
```suggestion
namespace picongpu::plugins::binning
```
and the corresponding change at the bottom.